dev/tools/shared/images.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "service_name": "agent-sandbox-controller",
+    "context_dir": ".",
+    "dockerfile": "Dockerfile"
+  },
+  {
+    "service_name": "chrome-sandbox",
+    "context_dir": "examples/chrome-sandbox",
+    "dockerfile": "examples/chrome-sandbox/Dockerfile"
+  },
+  {
+    "service_name": "python-runtime-sandbox",
+    "context_dir": "examples/python-runtime-sandbox",
+    "dockerfile": "examples/python-runtime-sandbox/Dockerfile"
+  },
+  {
+    "service_name": "sandbox_router",
```

Comment:
`-` seems preferred and more commonly used in the container ecosystem. It's also used in other images (consistency).